tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -189,8 +150,8 @@ var _ = ginkgov2.Describe("VM backup and restore tests", ginkgov2.Ordered, func(
 		},
 
 		ginkgov2.Entry("no-application CSI datamover backup and restore, CirrOS VM", ginkgov2.Label("virt"), VmBackupRestoreCase{
-			Source:          "cirros-dv",
-			SourceNamespace: "openshift-cnv",
+			Template:  "./sample-applications/virtual-machines/cirros-test/cirros-test.yaml",
+			InitDelay: 2 * time.Minute,
```

Comment:
why we need delay now with new approach?

Reply:
The VM gets marked running while kernel boot messages are still scrolling on the console, and I would rather start the backup when it's sitting idle like at the login screen. I don't like having to sleep to wait for the VM to get ready, but I don't have a better solution ready yet.

Reply:
fair enough, a comment about this is enough for me for now